frontend/src/redux/reducers/userReducer.reducer.js

```diff
@@ -0,0 +1,21 @@
+import { login, logout } from 'services/auth';
+
+const initialState = {
+  user: 0
```

Comment:
Here are we going to have more information from the user? or just the user token? if we will have more information I think it would be better if the initial state is 
```suggestion
  user: {}
```